web/impact/impact/urls.py

```diff
@@ -35,14 +35,18 @@
     IndexView,
     JWTCookieNameView,
 )
+from .views.general_view_set import EXCLUDE_MODELS
 
 accelerator_router = routers.DefaultRouter()
 simpleuser_router = routers.DefaultRouter()
 simpleuser_router.register('User', GeneralViewSet, base_name='User')
 
 for model in apps.get_models('accelerator'):
-    if model._meta.app_label == 'accelerator' and not model._meta.auto_created:
-        schema_router.register(model, url=model_name_to_snake(model.__name__))
+    if (model._meta.app_label == 'accelerator' and not
+            model._meta.auto_created and
+            model.__name__ not in EXCLUDE_MODELS):
```

Comment:
@mkiterian instead of the setting, for consistency, couldn't we just create a meta option that flags whether the model should be excluded?

Reply:
Thanks for the suggestion @kitaka . I've looked into the [meta options](https://docs.djangoproject.com/en/1.11/ref/models/options/) and there only a predefined set non of which we can use. Custom options cannot not be created [ref](https://code.djangoproject.com/ticket/5793).